pkgs/desktops/elementary/wingpanel-indicators/keyboard/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, elementary, pkgconfig, meson, ninja
+, substituteAll, vala, gtk3, granite, libxml2, wingpanel, libgee
+, xorg, libgnomekbd, gobjectIntrospection, defaultIconTheme, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "wingpanel-indicator-keyboard";
+  version = "2.1.1";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "elementary";
+    repo = pname;
+    rev = version;
+    sha256 = "0x0bdd9svw4sffx8pa0pqlh6mrj3fqp4mgrb4n7ys26k2w20ngnb";
+  };
+
+  passthru = {
+    updateScript = elementary.updateScript {
+      repoName = pname;
+    };
+  };
+
+  nativeBuildInputs = [
+    gobjectIntrospection
+    meson
+    ninja
+    libxml2
+    pkgconfig
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    defaultIconTheme
+    granite
+    gtk3
+    libgee
+    wingpanel
+  ];
+
+  patches = [
+    (substituteAll {
+      src = ./xkb.patch;
```

Comment:
Why not just use single `fix-paths.patch`? I find it more practical but feel free to keep it separate if you like it better.